### PR TITLE
Add test coverage for organization, location and env entities (BZ1479736)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_environment.py
......
----------------------------------------------------------------------
Ran 6 tests in 336.294s

OK
```

All other tests were executed using debugger and some manual interactions as BZs are in assigned state

### CODE REVIEW

File: tests/foreman/ui/test_location.py [hunk 1]
```diff
@@ -356,6 +358,45 @@ def test_positive_add_user(self):
...
+        :BZ: 1321543
+
+        :CaseLevel: Integration
+        """
+        loc_name = gen_string('alpha')
+        user = entities.User().create()
+        with Session(self) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            make_loc(session, name=loc_name)
+            self.location.update(loc_name, new_users=[user.login])
```
Comment (abalakh) [inline tests/foreman/ui/test_location.py]: Maybe i got the BZ wrong, but according to it, there was no initial location+user assignment, i.e. user was only associated via 'All Users' checkbox.
Your scenario is testing a bit different thing - it ensures user will remain associated regardless of 'All Users' checkbox state if he was associated before.

Comment (oshtaier) [reply tests/foreman/ui/test_location.py]: Look closer to 1479736 BZ, but basically I was looking into exactly thing that you described. If you like I can add 1479736 id to all BZs

Comment (abalakh) [reply tests/foreman/ui/test_location.py]: That BZ describes everything, thanks. This coverage is good but BZ1321543 requires a bit different scenario. My suggestion - leave this test as is, just replace BZ to 1479736 in docstring and add similar test for BZ1321543 specifically, which won't have user assigned initially, this way we'll ensure association appears after checking off 'All Users' checkbox, like BZ1321543 states.

**Comment by oshtaier** [comment]
@abalakh addressed

**Comment by oshtaier** [comment]
skipped
skipped
skipped

File: tests/foreman/ui/test_environment.py [hunk 2]
```diff
@@ -51,6 +52,40 @@ def test_positive_create_with_name(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_create_with_org(self):
+        """Create new environment with organization
```
Comment (ldjebran) [inline tests/foreman/ui/test_environment.py]: perhaps create_with_org_name and in the future we can create other similar function create_with_org_id

Comment (ldjebran) [reply tests/foreman/ui/test_environment.py]: sorry,  this is UI test  :)

**Pull request merged**